src/tests/ftest/util/ior_utils.py

```diff
@@ -33,7 +33,7 @@
 class IorFailed(Exception):
     """Raise if Ior failed."""
 
-
+# pylint: disable=attribute-defined-outside-init
```

Comment:
What was defined outside of `__init__`?  Can it not be defined in `__init__` rather than using this check disabler?

In any case the disabler should be re-enabled as soon as not needed so as not to apply to every instance of this check in the file.